src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -523,6 +523,9 @@
   <data name="ToolSettingsUnsupportedRunner" xml:space="preserve">
     <value>Command '{0}' uses unsupported runner '{1}'."</value>
   </data>
+  <data name="TopLevelPublishConfigurationMismatchError" xml:space="preserve">
+    <value>Multiple top-level projects in the given solution contain conflicting PublishRelease or PackRelease values. To use conflicting configurations, leverage a Directory.build.props file instead of setting flags that don't match.</value>
```

Comment:
I don't think so since `Directory.build.props` will apply to all projects. We would still generate the error if the values in the projects themselves still mismatch. I provided a fix.